test/keys/README.md

```diff
@@ -1,4 +1,7 @@
 Keys for tests generated by executing:
 
-kumactl generate signing-key --format=pem > samplekey.pem
-kumactl generate public-key --signing-key-path=samplekey.pem > publickey.pem
+kumactl generate signing-key --format=pem > sample-key-static-1.pem
+kumactl generate public-key --signing-key-path=sample-key-static-1.pem > sample-publickey-static-1.pem
```

Comment:
I believe this key is used by unit tests as well so renaming will blow those tests up.

btw. key itself has no motion of the mesh. It's also used in context of other signing keys. I'd just name this
```
samplekey.pem
samplekey-2.pem
```
or if you want to rename it
```
samplekey-1.pem
samplekey-2.pem
```

Reply:
Fair enough 🙂. I went with your first suggestion